src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/TypeAccessibilityCodeActionProvider.cs

```diff
@@ -142,14 +144,14 @@ internal sealed class TypeAccessibilityCodeActionProvider : ICSharpCodeActionPro
             }
         }
 
-        return typeAccessibilityCodeActions;
+        return typeAccessibilityCodeActions.ToImmutable();
     }
 
-    private static IEnumerable<RazorVSInternalCodeAction> ProcessCodeActionsVS(
+    private static ImmutableArray<RazorVSInternalCodeAction> ProcessCodeActionsVS(
         RazorCodeActionContext context,
-        IEnumerable<RazorVSInternalCodeAction> codeActions)
+        ImmutableArray<RazorVSInternalCodeAction> codeActions)
     {
-        var typeAccessibilityCodeActions = new List<RazorVSInternalCodeAction>(1);
```

Comment:
@DustinCampbell this had a list set to 1 and I'm not really sure why... I think I wouldn't need to do that here because that would just keep it on the stack?

Reply:
Super weird! I would just ignore the capacity in that case.